lambdas/cfn-s3objects-macro/template.yaml

```diff
@@ -0,0 +1,29 @@
+# From https://github.com/awslabs/aws-cloudformation-templates/tree/master/aws/services/CloudFormation/MacrosExamples/S3Objects
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
@zaro0508 Is this ^^ the only line that is different between the two PRs?

Reply:
no, i think i made some other changes as well.  I've verified that this is working in sandbox so i think we good to go